globalize.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activerecord', '>= 4.2.0', '< 4.3'
   s.add_dependency 'activemodel', '>= 4.2.0', '< 4.3'
+  s.add_dependency 'request_store', '~> 1.1'
```

Comment:
I see your point.
~> 1.0 means that less people would also have to update request_store from 1.0 to 1.1, if they already have it.
I changed it.